sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/PatternAnalyzer.java

```diff
@@ -37,7 +42,7 @@
      * Regular expression flags.
      */
     @JsonProperty(value = "flags")
-    private List<RegexFlags> flags;
+    private RegexFlags flags;
```

Comment:
it changed from list to a single RegexFlags?

Reply:
Correct, the actual type returned from the service is a single string property. This now aligns to what the service expects and returns